src/pybind/mgr/dashboard/frontend/cypress/integration/cluster/cluster-welcome-page.po.ts

```diff
@@ -9,11 +9,6 @@ export class CreateClusterWelcomePageHelper extends PageHelper {
   createCluster() {
     cy.get('cd-create-cluster').should('contain.text', 'Welcome to Ceph');
     cy.get('[name=create-cluster]').click();
-
-    cy.get('cd-dashboard').should('exist');
-    const notification = new NotificationSidebarPageHelper();
-    notification.open();
-    notification.getNotifications().should('contain', 'Cluster creation feature not implemented');
```

Comment:
IMHO, these should be removed in the add host e2e section. Review section is something that comes after the add host section, so that tests should go first and these should be removed from that test itself. Otherwise the add host is gonna fail.

Reply:
add host e2e is WIP , as I need to adopt some changes there, but I removed this as this PR was gonna get merged first so the changes required for the add the host section can be added later in it's e2e itself.